### ISSUE TITLE
Something went wrong while displaying this page (Chromedriver)

### ISSUE THREAD

**Issue by jadedtakesthiscereal** [maximedrn/opensea-automatic-bulk-upload-and-sale#279]
Running 1.7.1 successfully with coinbase wallet *most of the time*  the only full stop error I do run into is after 25-50 upload and sales the next browser page when loading to sell usually goes to an “uh oh blank screen with a reload button” (not the standard 404) and the wallet remains waiting in the small window.  If I am nearby and click reload myself it does continue.  If not I get this.
![55A554B5-20C4-477C-BC8C-9B1CF43326D7](https://user-images.githubusercontent.com/102177219/166124631-060aa37d-d368-4a97-acee-10e60f397d6b.jpeg)

**Comment by jadedtakesthiscereal** [comment]
After another 35 got screen I was mentioning in a pic 
![E08C5739-C56C-4D8C-B601-936D1AAD9158](https://user-images.githubusercontent.com/102177219/166126185-5d7cfd30-d6cb-44c3-8e70-5062421c9dbf.png)

**Comment by jadedtakesthiscereal** [comment]
Deletin temp files etc to see how far I can get.  It would go hundreds with no issues on metamask

**Comment by jadedtakesthiscereal** [comment]
.. follow up, deleted %temp% files, disk cleanup etc’d, restarted pc.. all the normal clear up memory routines and got to 33 and same page as above.  As stated my system does not do this with metamask.  But I have no idea why the other wallet would matter.

**Comment by maximedrn** [comment]
It comes from memory of your PC, how many GB RAM your PC has?

**Comment by jadedtakesthiscereal** [comment]
Just 16, I am just at a loss as to why my memory was not an issue before.

**Comment by maximedrn** [comment]
> Just 16, I am just at a loss as to why my memory was not an issue before.

I do not know, it's not your memory, it's Google Chrome :/
You can use MetaMask instead, Coinbase Wallet is not very useful because the bot works with a recovery phrase to link a wallet. So MetaMask can also use a Coinbase wallet.

**Comment by jadedtakesthiscereal** [comment]
Ok using metamask with my coinbase wallet info, will let you know how it goes on longer run.  (Was not aware I could run cb wallet through metamask, thank you for that information.)

**Comment by jadedtakesthiscereal** [comment]
Using 1.7.1 metamask with coinbase wallet credentials.. same memory error after 36 this time.  Chrome did have another update the other day, maybe something there changed memory usage.  Does not seem to be an issue with your program as it has worked effectively before and now only hiccups at that memory issue.  Close this if ya feel comfortable with that.

**Comment by maximedrn** [comment]
I think this comes from your computer for some reason. You can use MetaMask and Geckodriver, it does not use Chromium (based on Google Chrome) so this problem might not appear.

**Comment by jadedtakesthiscereal** [comment]
I do not know why but headless mode has never worked well on my pc.  When one of the versions had a geckodriver no headless mode option is the only time I have been able to use firefox.  Tried yesterday with 1.7.1 and after 15 minutes 1 nft had uploaded and not listed for sale and the screen was just at ‘sale of nft’ but never seemed to move.  I WAS ABLE to get my stuff up for this collection so this is NOT a rush ‘omg they need no headless option for geckodriver’.  Just might help me and others since now (maybe chromes newest update) 16gb of ram only gets me 30-40 uploads before browser shows full memory (though a simple reload click lets the bot continue,, is it possible to add a “if this screen appears click reload- type arguement”.  Again nothing I am like hurry up Maxime on :) have a great day.
